docs/infrastructure/telegram.rst

```diff
@@ -0,0 +1,108 @@
+########
+Telegram
+########
+
+`Telegram <https://telegram.org/>`__ is a messaging client similar to WhatsApp, GroupMe, Messenger, etc.
+This document explains the infrastructure and management of the RITlug Telegram group, `@ritlug`_.
+
+.. figure:: /_static/img/telegram/telegram-group-overview.png
+   :alt: Screenshot of the @ritlug Telegram group
+
+   Screenshot of the `@ritlug`_ Telegram group
+
+
+**********
+Background
+**********
+
+Today, the Telegram group is left online as a convenience to older members who choose to use it.
+However, it is not officially advertised or endorsed by RITlug eboard.
+
+Before switching to Slack, RITlug advertised a Telegram group and :doc:`IRC channel <irc-channel>`.
+After switching to Slack and setting up a bridge, RITlug only advertised the Slack.
+However, the Telegram group and IRC channel were kept online for convenience.
+The work to maintain a bridge across all groups was simple: run a bot and forget.
```

Comment:
I think we should link/explain how we are running the telegram bridge bot on this page.